misc/testerina/modules/testerina-runtime/src/main/java/org/ballerinalang/test/runtime/util/CodeCoverageUtils.java

```diff
@@ -81,14 +78,12 @@
      * @param destination path to extract the classes
      * @param orgName     org name of the project being executed
      * @param moduleName  name of the module being executed
-     * @param version     version of the module being executed
      * @throws NoSuchFileException if source file doesnt exist
      */
-    public static void unzipCompiledSource(Path source, Path destination, String orgName, String moduleName,
-                                           String version, boolean enableIncludesFilter,
```

Comment:
Don't we need the version also?

Reply:
It was used previously to create a structure and copy to bin path. Now it's not required.